README.md

```diff
@@ -40,7 +40,7 @@ If you are not sure, ask someone in the SIG where you initially circulated the i
 
 ## When to Create a New Enhancement Issue
 
-Create an issue here once you:
+Create an [issue](https://git.k8s.io/enhancements/ISSUE_TEMPLATE.md) in this repository once you:
```

Comment:
Can we link to the create issue link instead? https://github.com/kubernetes/enhancements/issues/new

This removes additional clicks on behalf of people wanting to create an issue.

Reply:
sure!

Reply:
Done PTAL

Reply:
Thank you! :)